dependency_support/com_github_alexforencich_verilog_axi/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 The XLS Authors
```

Comment:
I don't think there is a precedent for our project to import SystemVerilog third_party dependency, I'll have to figure out how to make this work.

Reply:
as we discussed, it might be worth exploring landing the verilog dependency in a separate PR (and merge first the cocotb tests that don't depend on it)